src/GNDStk/Tree/src/read.hpp

```diff
@@ -70,7 +70,7 @@ std::istream &read(
          // assume .xml
          // go through a temporary XML object to create the tree...
          const XML tmp(is);
-         if (not is.fail())
+         if (!is.fail())
```

Comment:
These changes were not necessary.

Reply:
Since these changes do not alter the intent of the code, this is a non-issue in my opinion so there's no need to call it out.